manifest.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "10up Scaffold",
+  "short_name": "10up",
```

Comment:
Let's just "10up Scaffold" here since that's what the CLI scans for and people can update it later during the project if need be